src/unix/mod.rs

```diff
@@ -143,6 +143,9 @@ cfg_if! {
     } else if #[cfg(target_os = "emscripten")] {
         #[link(name = "c")]
         extern {}
+    } else if #[cfg(all(target_vendor = "rumprun", target_os = "netbsd"))] {
+        #[link(name = "m")]
```

Comment:
Whoa, weird! If we're triggering bugs I guess it seems fine to leave this here. Could you add some comments to this effect though?